docs/content.go

```diff
@@ -192,8 +190,7 @@ Enterprise Performance Computing (EPC)`
 	CacheListUse   string = `list [list options...]`
 	CacheListShort string = `List your local Singularity cache`
 	CacheListLong  string = `
-  This will list you local cache: "${HOME}/.singularity/cache". The available cache
-  types are: library, oci, and blob.`
+  This will list you local cache stored at $HOME/.singularity/cache.`
```

Comment:
By default the cache directory is `$HOME/.singularity/cache`, but if `SINGULARITY_CACHEDIR` is set, the cache directory will change. Maybe this can be updated like:

```
This will list your local Singularity cache.
```

*Also `you` => `your`.*

Reply:
Ok, updated

Reply:
Here too:

```suggestion
  This will clean your local cache (stored at $HOME/.singularity/cache if SINGULARITY_CACHEDIR is not set).
  By default only oci blobs cache is cleaned, use '--all' to clean all cache.`
```

Reply:
@WestleyK Are suggesting to write `cache clean` description where `cache list` should be?

Reply:
Sorry, copy-paste error. Fixed now! :wink: